R/billingInfo.R

```diff
@@ -13,6 +13,10 @@ getBillingAccount <- function(billingAccountId, asDataFrame = TRUE) {
   
   billingInfo <- getResource(paste0("/billingAccounts/", billingAccountId), task = "Getting billing account info")
   billingInfo$id <- as.character(billingInfo$id)
+  if (length(billingInfo$addons) == 0) {
+    billingInfo$addons <- c()
```

Comment:
Did you test this? I believe that `c()` evaluates to `NULL`. `character(0)` will give you a zero-length string vector.